src/cljs/nr/utils.cljs

```diff
@@ -111,6 +112,16 @@
    (let [f (aget js/toastr toast-type)]
      (f msg))))
 
+(defn tr-non-game-toast
+  "Display a toast warning with the specified translation vector."
+  ([tr-vec toast-type] (tr-non-game-toast tr-vec toast-type nil))
+  ([tr-vec toast-type options] (tr-non-game-toast tr-vec nil toast-type options))
+  ([tr-vec tr-params toast-type options]
+   (set! (.-options js/toastr) (toastr-options options))
+   (let [f (aget js/toastr toast-type)
+         msg (rdom-server/render-to-string (tr-span tr-vec tr-params))]
```

Comment:
Should we be using `tr-span` here? I know that toasts need to be translated too, but seems... ill-advised to be rendering hiccup to string like this. maybe not? hard to say.